src/traces/funnelarea/attributes.js

```diff
@@ -0,0 +1,85 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var pieAttrs = require('../pie/attributes');
+var sunburstMarker = require('../sunburst/attributes').marker;
+var plotAttrs = require('../../plots/attributes');
+var domainAttrs = require('../../plots/domain').attributes;
+var hovertemplateAttrs = require('../../components/fx/hovertemplate_attributes');
+
+var extendFlat = require('../../lib/extend').extendFlat;
+
+module.exports = {
+    labels: pieAttrs.labels,
+    // equivalent of x0 and dx, if label is missing
+    label0: pieAttrs.label0,
+    dlabel: pieAttrs.dlabel,
+    values: pieAttrs.values,
+    marker: sunburstMarker,
```

Comment:
So, `funnelarea` traces like sunburst have a default `marker.line.width` of 1 and use `paper_color` to set the default `maker.line.color` ? I think that's ok, but I would prefer making this more explicit here:


```
    marker: {
        colors: pieAttrs.colors,
  
        line: {
            color: extendFlat({}, pieAtts.marker.line.color, {
                dflt: null,
                description: [
                    'Sets the color of the line enclosing each sector.',
                    'Defaults to the `paper_bgcolor` value.'
                ].join(' ')
            }),
            width: extendFlat({}, pieAtts.marker.line.width, {dflt: 1}),
            editType: 'calc'
        },
        editType: 'calc'
    }
```

That way both `sunburst` and `funnelarea` inherit from `pie`, and we don't need to make `funnelarea` inherit from `sunburst` too.

Reply:
Done in https://github.com/plotly/plotly.js/pull/3876/commits/316aff912a4cab9a6b899e86764977e0ee82766f.